device/routes/routeaddresses.yaml

```diff
@@ -0,0 +1,67 @@
+components:
+  schemas:
+    V4RouteAddress:
+      description: >-
+        A container for IPv4 route addresses.
+      type: object
+      required: [address]
+      properties:
+        address:
+          description: >-
+            The starting address of the network.
+          type: string
+          format: ipv4
+        prefix:
+          description: >-
+            The IPv4 network prefix length to be applied to the address. 
+          type: integer
+          default: 24
+          minimum: 0
+          maximum: 32
+        count:
+          description: >-
+            The total number of addresses in the range.
+          type: integer
+          default: 1
+          minimum: 1
+        step:
+          description: >-
+            Increments the network address prefixes within a route range 
+            where multiple routes are present. 
+            The value is incremented according to the Prefix Length and Step.
+          type: integer
+          default: 1
+          minimum: 1
+
+    V6RouteAddress:
+      description: >-
+        A container for IPv6 route addresses.
+      type: object
+      required: [address]
+      properties:
+        address:
+          description: >-
+            The starting address of the network.
+          type: string
+          format: ipv6
+        prefix:
+          description: >-
+            The IPv6 network prefix length to be applied to the address.
+          type: integer
+          default: 64
+          minimum: 0
+          maximum: 128
+        count:
```

Comment:
No maximum for count and step?

Reply:
The maximum value for count and step really depends on prefix length. As these are of type integers that max limit should be respected by the types.